virt/virtual_machines/advanced_vm_management/virt-specifying-nodes-for-vms.adoc

```diff
@@ -0,0 +1,32 @@
+[id="virt-specifying-nodes-for-vms"]
+= Specifying nodes for virtual machines
+include::modules/virt-document-attributes.adoc[]
+include::modules/common-attributes.adoc[]
+:context: virt-specifying-nodes-for-vms
+
+toc::[]
+
+You can place virtual machines (VMs) on specific nodes by using node placement rules.
+
+include::modules/virt-about-node-placement-vms.adoc[leveloffset=+1]
+
+[id="node-placement-virt-components_{context}"]
+=== About node placement for {VirtProductName} components
```

Comment:
Question: Can this xref be moved to the Additional resources section? Or maybe the information rewritten as a sentence that follows the previous content: "Learn more about specifying nodes for OpenShift Virtualization components."? The level 3 heading with a link feels a bit abrupt to me. I understand that this allows the link to show up in the assembly toc, so having a separate heading might be the way to go after all.

Reply:
I like your suggestion for changing this to a sentence. It feels like the right location for the link, but maybe it should be in both places just in case?